### PR TITLE
Tricks page phase3 modification trick review

### PR DESCRIPTION
**Pull request by tanguyguillo**
i had : warning: Cannot merge binary files: .DS_Store (HEAD vs. main)

I try to order the branch conflict (.gitignore : .DS_Store)

and i did that : 

Capture d’écran 2023-01-18 à 12 21 19

seems to be ok now to merge  [tricksPagePhase3-modificationTrick](https://github.com/tanguyguillo/snowtricks/tree/tricksPagePhase3-modificationTrick)

into main...  @tanguyguillo  @castroCrea

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -14,31 +14,47 @@
...
 # https://symfony.com/doc/current/best_practices.html#use-environment-variables-for-infrastructure-configuration
 
-###> symfony/webapp-pack ###
-MESSENGER_TRANSPORT_DSN=doctrine://default?auto_setup=0
-###< symfony/webapp-pack ###
-
 ###> symfony/framework-bundle ###
 APP_ENV=dev
-APP_SECRET=1e5342d4dce5dd1f686d55d943522dfc
+APP_SECRET=187b0d05c80b4ad52baf61c728a90548
```
Comment (castroCrea) [inline .env:19]: better to write here "!ChangeMe!" It is kind of a convention, to don't forget it on prod

image

File: src/Controller/TrickController.php [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+    /**
+     * deleteAdditionnalPicture : Delete additional picture from the server
+     * 
+     *  #[Route('/test/{argument}', name: 'app_tricks_test',)]
+     */
+    //#[Route('/test/{argument}', name: 'app_tricks_test',)]
+    public function test()
+    {
+        //$this->deleteAdditionnalPicture(153);
+    }
```
Comment (castroCrea) [inline src/Controller/TrickController.php:341]: Do you still need that?

**Review by castroCrea — APPROVED** [review]
Generally It looks good to me 🦾 

Some improvement
- You pushed some images I don't know if this is wanted.
- You have a lot of code comments, In general, we prefer to keep code small and don't keep useless code. But if you feel safer with it now keep it and remove it before delivering the project

**Comment by tanguyguillo** [comment]
Hi Paolo

*"Generally It looks good to me 🦾 ...*. "     : yes....

*Some improvement*

   - *You pushed some *images I don't* know if this is wanted.  -> *yes
   (logo and background)  and for the folder avatars (not dev yet) , for the
   tricks folder it's the reflect of my currents tricks. is it better to
   exclude ? i have just one default picture : main-picture.jpg, but i can put
   it on a another folder or make and exception? will be  not necessary for
   Datafixtures?

what it's used to do?

   - *You have a lot of code comments, In general, we prefer to keep code
   small and don't keep useless code. But if you feel safer with it now keep
   it and remove it before delivering the project. :* yes it's just for now
   (and for me), anyway i had to clean it for SymfonyInsight

I have some code duplication also, to correct that later...

thank you for your reviews

may i merge in main branch ?

Have a good day

tanguy

Le jeu. 19 janv. 2023 à 08:28, castroCrea ***@***.***> a
écrit :

> ***@***.**** approved this pull request.
>
> Generally It looks good to me 🦾
>
> Some improvement
>
>    - You pushed some images I don't know if this is wanted.
>    - You have a lot of code comments, In general, we prefer to keep code
>    small and don't keep useless code. But if you feel safer with it now keep
>    it and remove it before delivering the project
>
> ------------------------------
>
> In .env
> <https://github.com/tanguyguillo/snowtricks/pull/22#discussion_r1080884943>
> :
>
> >  ###> symfony/framework-bundle ###
>
>  APP_ENV=dev
>
> -APP_SECRET=1e5342d4dce5dd1f686d55d943522dfc
>
> +APP_SECRET=187b0d05c80b4ad52baf61c728a90548
>
>
> better to write here "!ChangeMe!" It is kind of a convention, to don't
> forget it on prod
>
> [image: image]
> <https://user-images.githubusercontent.com/20707343/213379813-76712ecc-84f1-4ffc-a107-bc72cadc8d02.png>
>

Ok i'll do that, on prod it's another secret :

PP_ENV=prod
APP_SECRET=c023.......

File: .env (hunk 1 above)
Comment (tanguyguillo) [reply .env]: did it

File: src/Controller/TrickController.php (hunk 2 above)
Comment (tanguyguillo) [reply src/Controller/TrickController.php:341]: no it's just for me for testing stuffs

**Pull request merged**